dll/win32/framedyn/WbemTime.cpp

```diff
@@ -0,0 +1,522 @@
+/*
+* PROJECT:     ReactOS Kernel
+* LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+* PURPOSE:     Implementation of WBEMTime and WBEMTimeSpan classes.
+* COPYRIGHT:   Copyright 2017 Jin Juanshi (2033537949@qq.com)
+*/
+
+#include <WbemTime.h>
+#include <stdio.h>
+#include <time.h>
+
+static __inline void time_t_to_FILETIME(time_t t, LPFILETIME pft)
+{
+    LONGLONG ll = Int32x32To64(t, 10000000) + 116444736000000000;
```

Comment:
What are these constructs (`10000000` and `116444736000000000` specifically)?

Reply:
Yep, hardcoded values must go away, and variables must have a human-readable name.

Reply:
I've used this sample code.
https://msdn.microsoft.com/en-us/library/windows/desktop/ms724228(v=vs.85).aspx

Reply:
According to [this blog](http://sunshine2k.blogspot.it/2014/08/where-does-116444736000000000-come-from.html) `116444736000000000` is actually an offset factor constant used as a FILETIME value for conversion. 

Therefor, this might explain something but I believe such magical constants like these must be explicitly stated with further comments inside the code. 

ReactOS developers would cringe a lot seeing magical constants and constructs without any kind of explanation what they're meant for.

Reply:
I've fixed :)